packages/ckeditor5-utils/tests/collection.js

```diff
@@ -161,15 +184,28 @@ describe( 'Collection', () => {
 			collection.add( {} );
 			expect( collection ).to.have.length( 2 );
 		} );
+	} );
+
+	describe( 'addMany()', () => {
```

Comment:
Here I would leave the old tests for `add()` - those are still valid and we do not care how they are handled internally.

A similar tests for `addMany()` are also needed. Kind duplicates code tests but we're adding new behavior so it would be nice to describe it with tests even though we're be testing similar things twice (because of implementation). Implementation might change but tests should stay the same.

Reply:
Resolved in 8cd7ff93.